### PR TITLE
Add meta EIP for Spurious Dragon

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

**Comment by pirapira** [comment]
This looks like the Spurious Dragon that I know.

File: EIPS/eip-draft_hfmeta_spuriousdragon.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+## Abstract
+
+This specifies the changes included in the hard fork named Spurious Dragon.
+
+## Specification
+
+- Codename: Spurious Dragon
+- Activation:
+  - Block >= 2,675,000 on Mainnet
```
Comment (chfast) [inline EIPS/eip-draft_hfmeta_spuriousdragon.md]: Shouldn't the activation be a single event expressed by `==`?

Comment (holgerd77) [inline EIPS/eip-draft_hfmeta_spuriousdragon.md]: I would find it extremely useful if these meta EIPs get merged. Always searching my way through the pull requests when looking for an overview of fork EIPs.

Comment (cdetrio) [inline EIPS/eip-draft_hfmeta_spuriousdragon.md]: @chfast `>=` is the convention

**Pull request merged**